xds/bootstrap/bootstrap.go

```diff
@@ -34,7 +34,7 @@ import (
 // registry is a map from credential type name to Credential builder.
 var registry = make(map[string]Credentials)
 
-// Credentials interface encapsulates a credential provider
+// Credentials interface encapsulates a creds builder
```

Comment:
Picking nits time: "a credentials.Bundle builder"?

I don't like using "creds" in external documentation, generally.  It feels too informal and random.

Reply:
Done.